lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -0,0 +1,122 @@
+                                                                files_to_update,
+                                                                files_to_update_without_prerelease_modifiers)
+        Helper::RevenuecatInternalHelper.attach_changelog_to_master(new_version_number, changelog_latest_path, changelog_path)
+        Helper::RevenuecatInternalHelper.commmit_changes_and_push_current_branch("Version bump for #{new_version_number}")
+        Helper::RevenuecatInternalHelper.create_release_pr(new_version_number, changelog, repo_name, github_pr_token)
+      end
+
+      def self.description
+        "Bumps sdk version, asks to update changelog and creates PR with changes."
+      end
+
+      def self.authors
+        ["Toni Rico"]
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :current_version,
+                                       description: "Current version of the sdk",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :changelog_latest_path,
+                                       description: "Path to CHANGELOG.latest.md file",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :changelog_path,
+                                       description: "Path to CHANGELOG.md file",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :files_to_update,
+                                       env_name: "RC_INTERNAL_FILES_TO_UPDATE_VERSION",
+                                       description: "Files that contain the version number and need to have it updated",
+                                       optional: false,
+                                       type: Array),
+          FastlaneCore::ConfigItem.new(key: :files_to_update_without_prerelease_modifiers,
+                                       env_name: "RC_INTERNAL_FILES_TO_UPDATE_VERSION_WITHOUT_PRERELEASE_MODIFIERS",
+                                       description: "Files that contain the version number without release modifiers and need to have it updated",
+                                       optional: true,
+                                       default_value: [],
+                                       type: Array),
+          FastlaneCore::ConfigItem.new(key: :repo_name,
+                                       env_name: "RC_INTERNAL_REPO_NAME",
+                                       description: "Name of the repo of the SDK",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :github_pr_token,
+                                       env_name: "GITHUB_PULL_REQUEST_API_TOKEN",
```

Comment:
I thought about renaming this env within the `RC_INTERNAL_` namespace as well... but I felt I could reuse the one that is used by the `create_pull_request` fastlane action... Lmk if you think I should rename this. It would also mean ios devs need to update their ENVs to use the new name.

Reply:
I'm cool with keeping this 😄